src/Mvc/Mvc.ApiExplorer/src/DefaultApiDescriptionProvider.cs

```diff
@@ -399,7 +398,7 @@ private IReadOnlyList<ApiRequestFormat> GetSupportedFormats(MediaTypeCollection
             {
                 contentTypes = new MediaTypeCollection
                 {
-                    (string)null,
+                    (string)null!,
```

Comment:
Fixing this was super nasty, so I'm going to assume no sane user would try and do this. API Explorer needs it specifically so that it can loop over a foreach.